definitions/pipelines/gathered_somatic_exome.cwl

```diff
@@ -129,6 +129,8 @@ inputs:
         secondaryFiles: [.tbi]
     output_dir: 
         type: string
+    somalier_vcf:
```

Comment:
@chrisamiller  Do you have the real input path for this so we can test it ?

Reply:
The test data one is in the new gzipped test data file, which is part of this PR.  The real one has been added here:
/gscmnt/gc2560/core/annotation_data/concordance_snps/GRC-human-build38_gnomad_exome_common_snps.vcf